learn/WebApi/GeoLocation/geolocation.js

```diff
@@ -22,7 +25,13 @@ function show(position) {
       `<div>lokasi mu disini: <br>latitude: ${position.coords.latitude} <br>longitude: ${position.coords.longitude}</div>`
     ); // mengisi data di pop-up
 }
+
+function error(err) {
+  navStatus.innerHTML = `ERROR: ${err.message} (${err.code})`;
```

Comment:
Buat benerin codacynya, coba diganti innerText

```suggestion
  navStatus.innerText = `ERROR: ${err.message} (${err.code})`;
```